frontend/src/components/uam/components/users-table/components/actions-cell/actions-cell.tsx

```diff
@@ -0,0 +1,25 @@
+import { Icon } from 'components/common/common';
+import { FC } from 'react';
+import { CellProps } from 'react-table';
+
+import {
+  UsersTableActionsProps,
+  UsersTableRow,
+} from '../../../../common/types/types';
```

Comment:
Imports should be flattened

Reply:
But these are not common imports - they are from `uam` directory - I am thinking more about moving them to the `users-table/common` - same for imports in `get-users-columns`